test/dshell/linker_flag_with_spaces.d

```diff
@@ -0,0 +1,17 @@
+import dshell;
+
+void main()
+{
+    if (OS == "windows" && MODEL == "32")
+    {
+        writeln("Skipping test when using Optlink.");
+        return;
+    }
+
+    const lib = "$OUTPUT_BASE/dir with spaces/b$LIBEXT";
+    Vars.set("lib", lib);
+
+    mkdirFor(lib);
```

Comment:
This creates a folder `dmd\test\$OUTPUT_BASE\dir with spaces`, no variable expansions happen.

Reply:
Right, thx; fixed.